asdf/commands/tests/data/frames0.asdf

```diff
@@ -6,35 +6,29 @@
 asdf_library: !core/software-1.0.0 {author: Space Telescope Science Institute, homepage: 'http://github.com/spacetelescope/asdf',
   name: asdf, version: 1.2.2.dev858}
 frames:
-- !wcs/celestial_frame-1.1.0
```

Comment:
I removed the frame tags because defaults in the frame-1.1.0 schema fail validation, and this change allows us to fill in defaults in nested objects that we were missing before.